sample-lnd.conf

```diff
@@ -270,6 +270,18 @@
 ; 65536. The profile can be access at: http://localhost:<PORT>/debug/pprof/.
 ; profile=
 
+; Enable a blocking profile to be obtained from the profiling port. A blocking
+; profile can show where goroutines are blocking (stuck on mutexes, I/O, etc).
+; This takes a value from 0 to 1, with 0 turning off the setting, and 1 sampling
+; every blocking event (it's a rate value).
+; blockingprofile=0
+
+; Enable a mutex profile to be obtained from the profiling port. A mutex ;
+profile can show where goroutines are blocked on mutexes, and which mutexes
```

Comment:
Incorrect comment, semicolon must be at the beginning of the line, otherwise this breaks the file.

Reply:
Ah, vim auto line folding messed it up, fixed!